input/tangy-checkboxes.js

```diff
@@ -108,14 +109,81 @@ class TangyCheckboxes extends PolymerElement {
         value: false,
         observer: 'reflect',
         reflectToAttribute: true
-      }
+      },
+      optionsListSource: {
+        type: String,
+        observer: 'reflect',
+        value: ''
+      },
+      optionsListProperties: {
+        type: String,
+        observer: 'reflect',
+        value: ''
+      },
+      optionsListExcludes: {
+        type: String,
+        observer: 'reflect',
+        value: ''
+      },
+      optionsListExcludeBy: {
+        type: String,
+        observer: 'reflect',
+        value: ''
+      },
     }
   }
 
+  get optionsList() {
+    return this._optionsList ? this._optionsList : undefined
+  }
+
+  set optionsList(optionsList) {
+    this._optionsList = optionsList
+  }
+
   connectedCallback() {
     super.connectedCallback()
-    this.render()
-    this.reflect()
+
+    if (this.optionsListSource) {
+      let that = this
+      const request = new XMLHttpRequest();
```

Comment:
When we get around to it, now that we have more than one input that uses XMLHttpRequest, we should start using the `axios` library to clean up these HTTP requests.